_security-analytics/sec-analytics-config/log-types.md

```diff
@@ -1,51 +1,79 @@
+- **Security** includes [WAF]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/waf/).
+- **System Activity** includes [Linux]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/linux/) and [Windows]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/windows/).
+- **Other** accounts for log types that don't fit into a specific category. For more information see [Other log types]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/other/).
+
+
+## Creating custom log types
+
+When you need to connect to a data source not supported by the standard log type, create a custom log type: 
+
+1. From the dashboard, select **OpenSearch Plugins** > **Security Analytics**, and then select **Detectors** > **Log types**.
+1. Select **Create log type**.
+1. Enter a name and, optionally, a description for the log type.
+   
+   The log type name supports characters a--z (lowercase), 0--9, hyphens, and underscores.
+   {: .note }
+   
+1. Select a category. The categories are listed in [Supported log types]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/log-types/).
+1. Select **Create log type** in the lower-right corner of the screen. The screen returns to the **Log types** page, and the new log type appears in the list. Note that the source for the new log type indicates **Custom**.
+
+## About field mappings
+
+The log type specified when creating dectectors determine which fields are available for mapping. For example, when "Windows logs" is selected as the log type, this parameter, along with the specific detection rules, determines the list of detection field names available for the mapping. Similarly, the selected data source determines the list of log source field names that are available for the mapping.
+
+Security Analytics uses prepackaged Sigma rules for detector creation. It can automatically map important fields for a specific log type to the corresponding fields in the Sigma rules. The **Field Mappings** section when [creating detectors]() presents a view of automatically mapped fields while also providing the option to customize, change, or add new field mappings. When a detector includes customized rules, you can follow this step to manually map detector rule field names to log source field names.
+
+Because the system has the ability to automatically map field names, manually mapping the fields when ECS fields already exist in your document is optional. However, detector rules require some mappings in order to operate, depending on the detector rule. The more fields that can be mapped between detector fields and log source fields, the greater the accuracy of generated findings.
```

Comment:
```suggestion
Because the system can automatically map field names, manually mapping the fields when `ecs` fields exist in your document is optional. Detector rules, however, require certain mappings in order to operate. These mapping depend on the detector rule. The more fields that can be mapped between detector fields and log source fields, the greater the accuracy of generated findings.
```